jd4/config.py

```diff
@@ -33,6 +38,14 @@ def do_save_config():
 async def save_cookies():
     await get_event_loop().run_in_executor(None, _save_cookie_function)
 
+def try_get_int(key, default=0):
```

Comment:
Please remove this function. Also please don't do type conversion, just store an "int" in yaml.